pkg/validation/projects.go

```diff
@@ -0,0 +1,59 @@
+package validation
+
+import (
+	"fmt"
+
+	"github.com/devfile/api/v2/pkg/apis/workspaces/v1alpha2"
+)
+
+// ValidateStarterProjects checks if starter project has only one remote configured
+// and if the checkout remote matches the renote configured
+func ValidateStarterProjects(starterProject v1alpha2.StarterProject) (err error) {
```

Comment:
I think using a named return here is less clear than returning `nil` explicitly; the created value (`err`) is never referenced.